pkg/target/manager_test.go

```diff
@@ -462,6 +478,24 @@ var _ = Describe("Manager", func() {
 		assertTargetProvider(targetProvider, t)
 	})
 
+	It("should be able to unset selected control plane", func() {
+		t := target.NewTarget(gardenName, prod1Project.Name, "", prod1AmbiguousShoot.Name).WithControlPlane(true)
+		manager, targetProvider := createTestManager(t, *cfg, clientProvider, kubeconfigCache)
+
+		Expect(manager.UnsetTargetControlPlane()).Should(Equal(prod1AmbiguousShoot.Name))
+		assertTargetProvider(targetProvider, target.NewTarget(gardenName, prod1Project.Name, "", prod1AmbiguousShoot.Name))
+	})
+
+	It("should fail if no control plane targeted", func() {
```

Comment:
Shouldn't this be idempotent and not return an error?

Reply:
All other unset operations return an error in this case. E.g. `gctl target unset shoot` also fails if no shoot is set. We would need to change it for these cases, too.

Reply:
Ok. I would not return error in all these cases but more important is a consistent behavior.